responsibleai/responsibleai/_tools/causal/causal_result.py

```diff
@@ -50,6 +50,13 @@ def __init__(
         self.local_effects = local_effects
         self.policies = policies
 
+    @property
+    def is_computed(self):
```

Comment:
not sure if you want to add a doc here, it's not necessary since it's a private function, but it seems like we've been trying to document even the private functions lately

Reply:
Done.